Suppress switch fallthrough warning

philihp opened the pull request:
In doing a clean build, I encountered the following warning. The added annotation will suppress this warning.

```
compile:
    [mkdir] Created dir: /Users/philihp/work/LX/examples/LXHeadless/bin
    [javac] Compiling 211 source files to /Users/philihp/work/LX/examples/LXHeadless/bin
    [javac] /Users/philihp/work/LX/src/heronarts/lx/parameter/LXParameter.java:70: warning: [fallthrough] possible fall-through into case
    [javac]       case MILLISECONDS:
    [javac]       ^
    [javac] 1 warning
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/output/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/midi/remote/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/audio/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/midi/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/color/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/parameter/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/pattern/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/effect/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/model/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/modulator/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/transform/package-info.class
      [jar] Building jar: /Users/philihp/work/LX/examples/LXHeadless/bin/LXHeadless.jar
```

mcslee commented:
I just barely beat you to this one - fixed here: https://github.com/heronarts/LX/commit/f8bb184acf0cd489770761601c31852649f274e9

philihp committed:
Suppress switch fallthrough warning

In doing a clean build, I encountered the following warning. From the
comments near this line, this looks okay. The added annotation will
suppress this warning.

```
compile:
    [mkdir] Created dir: /Users/philihp/work/LX/examples/LXHeadless/bin
    [javac] Compiling 211 source files to /Users/philihp/work/LX/examples/LXHeadless/bin
    [javac] /Users/philihp/work/LX/src/heronarts/lx/parameter/LXParameter.java:70: warning: [fallthrough] possible fall-through into case
    [javac]       case MILLISECONDS:
    [javac]       ^
    [javac] 1 warning
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/output/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/midi/remote/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/audio/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/midi/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/color/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/parameter/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/pattern/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/effect/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/model/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/modulator/package-info.class
    [javac] Creating empty /Users/philihp/work/LX/examples/LXHeadless/bin/heronarts/lx/transform/package-info.class
      [jar] Building jar: /Users/philihp/work/LX/examples/LXHeadless/bin/LXHeadless.jar
```

mcslee commented:
Thanks for fixing. Funny, I had actually just removed this recently because after updating Eclipse, it was complaining that this annotation would be ignored. But I am definitely also seeing this warning on my own CLI ant build, so I'm happy to restore this and go figure out what's wrong with my Eclipse configuration...

Sorry for the annoyance. Appreciate it!
